modules/cms/classes/Controller.php

```diff
@@ -256,7 +266,13 @@ public function run($url = '/')
             return $result;
         }
 
-        return Response::make($result, $this->statusCode);
+        $response = Response::make($result, $this->statusCode);
+
+        if (Config::get('cms.enableCsrfProtection')) {
+            $this->addCsrfCookie($response);
```

Comment:
@daftspunk Should we perhaps call this method `addXsrfCookie` to make it more clear what it is?